jetty-ee10/jetty-ee10-osgi/test-jetty-ee10-osgi/src/test/java/org/eclipse/jetty/ee10/osgi/test/TestJettyOSGiBootHTTP2JDK9.java

```diff
@@ -45,6 +46,7 @@
 import static org.junit.Assert.assertTrue;
 import static org.ops4j.pax.exam.CoreOptions.mavenBundle;
 
+@Ignore
```

Comment:
You're going to need to be very careful when you merge this through to jetty-12.1, where some of these tests that you've added `@Ignore`s for here are in fact working and _not_ ignored in 12.1.

Reply:
Oh yeah, this will give me more to fix when merging :) 
when starting this I wanted to enable those again here but figure out it was the case for 12.1 so no need to do it for 12.x